packages/stateofjs/lib/components/survey/questions/FormSubmit.jsx

```diff
@@ -51,6 +53,7 @@ const FormSubmit = ({
           </Components.LoadingButton>
         ) : readOnly ? null : (
           <Components.LoadingButton
+            title={intl.formatMessage({ id: 'general.finish_survey' })}            
```

Comment:
```
    - key: general.finish_survey
      t: Finish Survey »
```

@SachaG I think we need to remove the french quote from locale file, and add it to the code instead. Any objections?

Reply:
I think you're probably right, unless some locales would want to use another symbol instead? Or maybe it could be in CSS instead of code, and we could give people a way to add their own locale-specific stylesheets…

Reply:
I don't think it depends on the culture, it's in fact an icon pointing the direction, no more than that.